tests/unit/python/foglamp/plugins/south/test_http_south.py

```diff
@@ -46,6 +44,7 @@ class TestHttpSouthIngest(object):
     """Unit tests foglamp.plugins.south.http_south.http_south.HttpSouthIngest
     """
 
+    @pytest.mark.asyncio
     async def test_post_sensor_reading_ok(self, event_loop):
```

Comment:
need to fix this failure in **suite** only! Neither earlier nor now its failing in as a module test or single def test!